sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -606,15 +606,15 @@ object GpuOverrides extends Logging {
    * @return true if it is allowed else false
    */
   def isSupportedType(dataType: DataType,
-      allowNull: Boolean = false,
-      allowDecimal: Boolean = false,
-      allowBinary: Boolean = false,
+      allowNull: Boolean = true,
+      allowDecimal: Boolean = true,
+      allowBinary: Boolean = true,
       allowCalendarInterval: Boolean = false,
-      allowArray: Boolean = false,
```

Comment:
This is being called by more than just InternalColumnarRddConverter.  HostColumnarToGpu is leveraging this, and it does not support some of the types being enabled here.